[Woo POS] Display store details from POS settings in POS refunded order HTML and plaintext email templates with misc improvements in POS emails

jaclync opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

<!-- For bug fixes: If known, please provide links to help with traceability and escape analysis. -->
<!-- Please include a link to the issue of the bug being fixed, if one doesn't exist please create it. -->
<!-- If the PR that introduced the bug is known, please also add its link below. -->

Closes WOOMOB-464

This pull request displays store details from Point of Sale (POS) settings in POS refunded order email HTML/plaintext templates similar to the changes for the POS completed order email in https://github.com/woocommerce/woocommerce/pull/57848. Additionally, there are some minor improvements/fixes to POS emails that I included in this PR:

- Zero change due amount (like `'0'` value in the metadata `_cash_change_amount` field) is now displayed in POS completed/refunded order emails.
- A hidden PHP error in POS completed order email class is fixed in https://github.com/woocommerce/woocommerce/pull/58012/commits/4a80e150ce9f59ff8c3aeb6c415340a9fe55a8d7.
- In WC Settings > Emails > POS completed order, the "Enabled/Disabled" toggle field is removed in https://github.com/woocommerce/woocommerce/pull/58012/commits/dac32a4ad87454f83b093db94fff664f68a9ea40 because this email is manually sent via the API. Othrewise, `/wc/v3/orders/$orderID/actions/send_email` for this email template still succeeds while the email is not actually sent. The other manually sent core template, customer invoive, also excludes this toggle.

### Enhancements to POS Email Templates:
* Updated the POS refunded order email templates (both HTML and plaintext) to include store details such as name, email, phone number, address, and refund/returns policy. These details are fetched from POS settings with the same default values. (`customer-pos-refunded-order.php`, `plain/customer-pos-refunded-order.php`) [[1]](diffhunk://#diff-6cdab2dae33ab0ddff79b7eb80f0d93a514db72c73734f3b30380f3c6cc260f0L32-R57) [[2]](diffhunk://#diff-6cdab2dae33ab0ddff79b7eb80f0d93a514db72c73734f3b30380f3c6cc260f0R97-R126) [[3]](diffhunk://#diff-1878abee40d83a6ffc34fa2b31bfbba0136071562d74f383eed6529600e37b1dR28-R41) [[4]](diffhunk://#diff-1878abee40d83a6ffc34fa2b31bfbba0136071562d74f383eed6529600e37b1dR82-R109)
* Modified email subject lines to include the store name instead of the site title for better clarity. (`class-wc-email-customer-pos-refunded-order.php`)

### Improvements to Email Functionality:
* Added a new method `init_form_fields` to override email settings form fields for manually sent emails, removing the enabled/disabled field. (`class-wc-email-customer-pos-completed-order.php`)
* Introduced methods to fetch POS-specific settings such as store name, email, phone number, address, and refund policy. (`class-wc-email-customer-pos-refunded-order.php`)

### Zero Change Due Amount Fix:
* Adjusted logic for handling cash payment change amounts in email totals to ensure proper formatting when the amount is explicitly set to zero. (`class-wc-email-customer-pos-completed-order.php`, `class-wc-email-customer-pos-refunded-order.php`) [[1]](diffhunk://#diff-d406aebcb98b0a2e79fae478673e016dae6f246fb24f5bf7402e027dcdb86004R281-L233) [[2]](diffhunk://#diff-6cea077827b80518b9b5d3724f72afd8ad9f055d53d7410bdef3eb04ef6fd555R453-L452)

### Updates to Test Cases:
* Updated test cases to validate the new behavior for cash payment change amounts, ensuring correct handling of zero values. (`class-wc-email-customer-pos-completed-order-test.php`, `class-wc-email-customer-pos-refunded-order-test.php`) [[1]](diffhunk://#diff-972c3894269c6b1c2c56b2ce836d23e17f1b2fb9b581927c955c74c3db506b35L31-R31) [[2]](diffhunk://#diff-972c3894269c6b1c2c56b2ce836d23e17f1b2fb9b581927c955c74c3db506b35L40-R40) [[3]](diffhunk://#diff-b580fdf7d6dd326cfa0019c0e4d39397b7f21911aefae813d9107c900039de69L31-R31) [[4]](diffhunk://#diff-b580fdf7d6dd326cfa0019c0e4d39397b7f21911aefae813d9107c900039de69L40-R40)

### Screenshots or screen recordings:

<!-- If this PR includes UI changes, please provide screenshots or a screen recording for clarity. -->
<!-- This section can be removed if not applicable. -->

POS completed order email:

cash payment email | email settings (Enabled/Disabled toggle removed)
-- | --
<img width="560" alt="Screenshot 2025-05-15 at 2 08 22 PM" src="https://github.com/user-attachments/assets/d33b48df-0acc-417b-b1b6-b73e751d78a3" /> | <img width="636" alt="Screenshot 2025-05-15 at 2 09 27 PM" src="https://github.com/user-attachments/assets/7a94c87b-1c6b-4c35-ab3d-891aafe8ceef" />

POS refunded email:

\ | beginning | order totals | below order totals
-- | -- | -- | --
partial refund | <img width="562" alt="Screenshot 2025-05-19 at 4 41 00 PM" src="https://github.com/user-attachments/assets/c1a033de-a53e-4e27-bacb-139605b838e1" /> | <img width="566" alt="Screenshot 2025-05-19 at 4 41 28 PM" src="https://github.com/user-attachments/assets/b6e00690-282d-4fe1-adca-2a97469b60f1" /> | <img width="561" alt="Screenshot 2025-05-19 at 4 41 41 PM" src="https://github.com/user-attachments/assets/d3775e69-ce0e-4a24-93f0-3960237bbf83" />
full refund | <img width="563" alt="Screenshot 2025-05-19 at 4 42 09 PM" src="https://github.com/user-attachments/assets/19a4516d-7b44-45ae-a4b0-b01468d920c8" /> | <img width="562" alt="Screenshot 2025-05-19 at 4 42 22 PM" src="https://github.com/user-attachments/assets/f7c5f7da-f32c-40de-b435-96f9f44ec041" /> | <img width="561" alt="Screenshot 2025-05-19 at 4 42 29 PM" src="https://github.com/user-attachments/assets/89933cf0-608e-49dc-a3a3-00530f011bbc" />

<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Prerequisites: the store has a way to see email logs (like with [WP Mail Logging plugin](https://wordpress.org/plugins/wp-mail-logging/)) or send emails.

- Run PR branch in development environment (e.g. `pnpm run --filter="@woocommerce/plugin-woocommerce" watch:build`), or test with Woo Beta Tester plugin and enable the `point-of-sale` admin feature flag (example p1746734640520419/1746734508.133719-slack-C070SJRA8DP)
- Make sure the block email editor feature flag is disabled in WC settings > Advanced > Features
- Go to WC settings > Emails > POS completed order --> there should not be a "Enabled/Disabled" toggle field as in other templates except of customer invoice
- Go to WC settings > Point of Sale tab, note the values set here
- Make an API request to create a POS order ("created_via": "pos-rest-api") with a billing email, metadata with 0 change due amount, and a line item with quantity 2 for testing partial/full refunds --> the request should succeed with an order ID in the response

<details>
<summary>Click to see example request body</summary>

```json
{
    "status": "completed",
    "currency": "USD",
    "billing": {
        "first_name": "",
        "last_name": "",
        "company": "",
        "address_1": "",
        "address_2": "",
        "city": "",
        "state": "",
        "postcode": "",
        "country": "",
        "email": "test@email.co",
        "phone": ""
    },
    "meta_data": [
        {
            "key": "_wc_order_attribution_source_type",
            "id": 0,
            "value": "mobile_app"
        },
        {
            "key": "_cash_change_amount",
            "value": "0"
        },
        {
            "key": "_charge_id",
            "value": "auth2j9wq8"
        }
    ],
    "line_items": [
        {
            "id": 0,
            "product_id": 40,
            "quantity": 2
        }
    ],
    "created_via": "pos-rest-api"
}
```

</details>

- With the ID of the created order, make a request to send POS completed order email `POST /wc/v3/orders/$orderId/actions/send_email` with request body `{"template_id": "customer_pos_completed_order"}` --> it should succeed with a message like "Email template &quot;POS completed order&quot; sent to ...". In email logs or if the store can send email, check the email that was just sent. The email should have a zero change due row in the order totals section
- For the same order ID, issue a partial refund in the app or core --> In email logs or if the store can send email, check the email that was just sent. The email should mention "partial refund" in the subject and heading. The email should not display the cash change due amount. The email should have a zero change due row in the order totals section. Below the order summary, there should be a section with heading that shows the store name from POS settings, and email/phone/address below when available in POS settings. If refund & returns policy is set in POS settings, there should be a section below the store details with a heading and the policy text
- Try adding/removing/updating details in POS settings, and issue the remaining amount for refund in the app or core --> The email should mention "refund" (instead of "partial refund") in the subject and heading. The email should not display the cash change due amount. The email should have a zero change due row in the order totals section. Below the order summary, there should be a section with heading that shows the store name from POS settings, and email/phone/address below when available in POS settings. If refund & returns policy is set in POS settings, there should be a section below the store details with a heading and the policy text
- Feel free to repeat the steps above with different POS settings

<!-- End testing instructions -->

### Testing that has already taken place:

<!-- Detail any testing that has already been conducted. -->
<!-- Include environment details such as hosting type, plugins, theme, store size, store age, and relevant settings. -->
<!-- Mention any analysis performed, such as assessing potential impacts on environment attributes and other plugins, performance profiling, or LLM/AI-based analysis. -->
<!-- Within the testing details you provide, please ensure that no sensitive information (such as API keys, passwords, user data, etc.) is included in this public pull request. -->

- [x] @jaclync tests that core refund order email is sent for non-POS orders.
- [x] @jaclync tests that when POS settings are not set (simulated by deleting options in database), default settings values are shown in the refund email

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box below and supplying data. -->
<!-- It will trigger the 'Add changelog to PR' CI job to create and push the entry into the branch. -->

<!-- Due to org permissions, the job might fail for PRs crated from a fork under GitHub organizations. Possible solutions: -->
<!-- * Create entry manually with `pnpm --filter='@woocommerce/plugin-woocommerce' changelog add` and push it into the branch (replace `@woocommerce/plugin-woocommerce` with package name from nearest `package.json` file) -->
<!-- * Create entry from supplied PR data and push it automatically `pnpm utils changefile pr-number-here -o github-org-name-here` -->

-   [ ] Automatically create a changelog entry from the details below.

<!-- If no changelog entry is required for this PR, you can specify that below and provide a comment explaining why. This cannot be used if you selected the option to automatically create a changelog entry above. -->

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

<!-- Choose only one -->

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment <!-- If your Pull Request doesn't require a changelog entry, a comment explaining why is required instead -->

</details>

jaclync committed:
POS refunded order email: update HTML and plaintext templates with details from POS settings and copy updates.

jaclync committed:
WC_Email_Customer_POS_Completed_Order: fix misplaced order check that results in PHP error in logs.

jaclync committed:
WC_Email_Customer_POS_Refunded_Order: add test cases for store details from POS settings.

jaclync committed:
Display zero change due amount when the meta data value is `0` that is skipped in `empty` check.

jaclync committed:
WC_Email_Customer_POS_Completed_Order: remove enabled/disabled toggle for completed order email as it is manually sent. Otherwise, the `send_email` action succeeds without an email sent.

jaclync committed:
Fix lint errors.

jaclync committed:
Add changelog.

jaclync committed:
Update core version in updated refunded order templates.

jaclync commented:
Hi @joshheald 👋 as mentioned in https://github.com/woocommerce/woocommerce/pull/57848#issuecomment-2882001479, I'm experimenting the review process by having your review before requesting a review from the Flux team after approval. Feel free to share any thoughts on this!

joshheald commented:
> it should succeed with a message like "Email template "POS completed order" sent to ...". 

Is this message pre-existing, or something you've added? The message I get looks like this:

```
{
    "message":"Email template &quot;POS completed order&quot; sent to {EMAIL}, via RapidAPI\/4.2.8 (Macintosh; OS X\/15.4.0) GCDHTTPRequest."
}
```

If it's new, just wondering:
- why include the user agent?
- why HTML encode the quotation marks, rather than escape them?

Not a big issue at all, and I assume it's pre-existing :)

joshheald commented on plugins/woocommerce/includes/emails/class-wc-email-customer-pos-refunded-order.php:
```diff
@@ -75,10 +76,13 @@ public function __construct() {
 		 * @return string
 		 */
 		public function get_default_subject( $partial = false ) {
+			$store_name = $this->get_pos_store_name();
```
There's still some consistency improvements to make with the store name:

![email header screenshot](https://github.com/user-attachments/assets/9a57efc9-e70f-460b-a5a8-1b141b589af0)

1 is from the `From name` setting in the Email sender options (best to keep this as is)
2 is from `Settings > WooCommerce > Point of Sale > Store name`
3 is from `Settings > General > Site Title`

I think we should avoid using 3

jaclync replied:
Great catch on the various store names.

> 1 is from the From name setting in the Email sender options (best to keep this as is)

I agree that it's best not to change it.

> 3 is from `Settings > General > Site Title`

This one is tricky, because this site title is shown in a shared email header template that not only includes the blog name but also other HTML setup that I think it'd best to keep for POS emails. This email header is triggered in all core emails.

https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/templates/emails/email-header.php#L64

I opened a separate issue WOOPRD-523 to update he shared email header template to allow passing a different store name in POS emails. I tried an initial pass of this change and it's not trivial.

joshheald commented on plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php:
```diff
@@ -228,9 +277,9 @@ public function add_unit_price( $item_id, $item, $order ) {
-			$cash_payment_change_due_amount           = $order->get_meta( '_cash_change_amount', true );
-			$formatted_cash_payment_change_due_amount = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
-			if ( ! empty( $cash_payment_change_due_amount ) ) {
+			$cash_payment_change_due_amount = $order->get_meta( '_cash_change_amount', true );
+			if ( '' !== $cash_payment_change_due_amount ) {
+				$formatted_cash_payment_change_due_amount     = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
```
![change in refund email](https://github.com/user-attachments/assets/9999bd5c-9102-4cc4-869b-882c7d593653)

I don't think we should display change in refund emails, 0 or otherwise.

When you get a refund, you just get your previous payment back... you don't get change. The figure we're using for change relates to the original purchase, but showing it here is a little confusing – is the shopper supposed to give change to the merchant? 😉

jaclync replied:
Good observation & suggestion! I agree that it's confusing including this in the refund emails as the total amount changed. I removed this row for refund emails in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.

joshheald commented on plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php:
```diff
@@ -67,7 +68,7 @@ public function trigger( $order_id, $template_id ) {
 
 			$this->setup_locale();
 
-			if ( $order_id && ! is_a( $order, 'WC_Order' ) ) {
```
What was the reason for the WC_order check? Can there be other kinds?

jaclync replied:
Other core emails check for this so I thought it's best to keep the same check. But the deleted check here is too early, the same check below is the correct one.

joshheald commented on plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php:
```diff
@@ -188,6 +189,54 @@ private function enable_order_email_actions_for_pos_orders() {
 		}
 
+		/**
+		 * Override settings form fields to remove the enbaled/disabled field as the email is manually sent.
```
```suggestion
		 * Override settings form fields to remove the enabled/disabled field as the email is manually sent.
```

joshheald replied:
More a question for the future Flux reviewer, really... is there a better way to do this than fully specifying all the fields?

For example, can we add an action which updates `form_fields` to remove the enabled/disabled flag?

jaclync replied:
Fixed typo in https://github.com/woocommerce/woocommerce/pull/58012/commits/3d18c01c5d43d1c1fead86ff48d78e5b0bddab9c.

> More a question for the future Flux reviewer, really... is there a better way to do this than fully specifying all the fields?

I followed the same approach as in the only manually triggered core email "customer invoice":

https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/emails/class-wc-email-customer-invoice.php#L198-L257

Will tag the Flux reviewer about this question.

jaclync replied:
Tagging @adimoldovan for the question about a better way to hide certain default email form fields.

adimoldovan replied:
> Tagging @adimoldovan for the question about a better way to hide certain default email form fields.

I'm removing myself and adding @albarin as reviewer from Flux for a more experienced set of eyes on this PR.

joshheald commented on plugins/woocommerce/includes/emails/class-wc-email-customer-pos-refunded-order.php:
```diff
@@ -447,9 +449,9 @@ private function register_refund_email_triggers() {
-			$cash_payment_change_due_amount           = $order->get_meta( '_cash_change_amount', true );
-			$formatted_cash_payment_change_due_amount = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
-			if ( ! empty( $cash_payment_change_due_amount ) ) {
+			$cash_payment_change_due_amount = $order->get_meta( '_cash_change_amount', true );
```
As above

jaclync replied:
Removed in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.

joshheald commented on plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-completed-order-test.php:
```diff
@@ -28,7 +28,7 @@ public function setUp(): void {
 		// Given order with cash payment change amount.
 		$order = OrderHelper::create_order();
-		$order->add_meta_data( '_cash_change_amount', '5.00' );
+		$order->add_meta_data( '_cash_change_amount', '0' );
```
Why change the value in this test? Perhaps it would be worth having a separate test for 0 change due?

jaclync replied:
Mostly to reduce the number of diffs by keeping just the edge case. The PR is still in manageable size, so I updated for the zero amount case to be a separate case in https://github.com/woocommerce/woocommerce/pull/58012/commits/1e7a236edad3d05f8e5d8f1bc245d38a88b5840d.

joshheald commented on plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-refunded-order-test.php:
```diff
@@ -37,7 +37,7 @@ public function test_order_item_totals_adds_formatted_cash_change_due_amount_to_
 		// Then cash payment change due amount is set and formatted correctly.
 		$this->assertArrayHasKey( 'cash_payment_change_due_amount', $totals );
-		$this->assertEquals( wc_price( '5.00', array( 'currency' => $order->get_currency() ) ), $totals['cash_payment_change_due_amount']['value'] );
+		$this->assertEquals( wc_price( '0', array( 'currency' => $order->get_currency() ) ), $totals['cash_payment_change_due_amount']['value'] );
```
This probably needs to check that we _don't_ add the change for the refund emails... even if it's non-zero.

jaclync replied:
Test cases updated in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.

joshheald reviewed (approved):
Generally looks good, and tests according to instructions, well done 👍 

A few questions and suggestions in-line about:

- Store name consistency
- Whether we should show change in refund emails
- If there's some way to avoid totally re-specifying fields to remove the toggle – it doesn't feel very DRY right now, but maybe it's the only/right way.

Am I right in thinking that block email support is still to come?

jaclync commented:
> > it should succeed with a message like "Email template "POS completed order" sent to ...".
> 
> Is this message pre-existing, or something you've added? The message I get looks like this:
> 
> ```
> {
>     "message":"Email template &quot;POS completed order&quot; sent to {EMAIL}, via RapidAPI\/4.2.8 (Macintosh; OS X\/15.4.0) GCDHTTPRequest."
> }
> ```
> 
> If it's new, just wondering:
> 
> * why include the user agent?
> * why HTML encode the quotation marks, rather than escape them?
> 
> Not a big issue at all, and I assume it's pre-existing :)

This message in the `send_email` response is pre-existing, and I don't know the answers. In the mobile apps where we make this request, we don't use this message and just make sure the request succeeds. From a quick search, it was implemented in [this PR](https://github.com/woocommerce/woocommerce/pull/54597/files#diff-4fb07ec6087391be1cece27f70c8a4290cd1ad843a7e15e1552d4c9582824abcR503-R504) but I didn't see feedback on this message string.

jaclync committed:
Merge branch 'trunk' into feat/WOOMOB-464-pos-settings-in-refunded-emails

jaclync committed:
Remove cash payment change due amount in POS refund emails to avoid confusion in UI/UX.

jaclync committed:
Fix typo in comment.

jaclync committed:
Add cash payment change due amount zero case as a separate test case.

jaclync commented:
Hi @adimoldovan, the mobile team has done testing on our side with the first pass of review, feel free to focus on reviewing the code. Thanks!

jaclync committed:
Remove `is_enabled` check for sending email now that the enabled/disabled toggle is removed from form fields.

jaclync committed:
Merge branch 'trunk' into feat/WOOMOB-464-pos-settings-in-refunded-emails

jaclync commented:
@albarin shared with me that the email is no longer sent after switching from trunk to PR branch, and I was able to reproduce it by removing the `wp_options` table row where `option_name = 'woocommerce_customer_pos_completed_order_settings'`. The email is not sent anymore because of [the `is_enabled` check](https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php#L81), and the removal of the enabled/disabled toggle from the form fields in https://github.com/woocommerce/woocommerce/pull/58012/commits/dac32a4ad87454f83b093db94fff664f68a9ea40 in this PR. When email settings are not stored in the database (like the row I deleted), [`get_option` falls back](https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php#L301) to the [default values from form fields](https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php#L285-L286). Now that the [default `enabled` field](https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/emails/class-wc-email.php#L990-L995) is removed, it's no longer getting the default value `yes` and thus the email is not being sent.

In https://github.com/woocommerce/woocommerce/pull/58012/commits/01479459d68f084f65b69e6cba06d80290bb983c, I removed the `is_enabled` check now that there's no enabled/disabled toggle. The core email "customer invoice" that is also manually sent [doesn't have the `is_enabled` check](https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/emails/class-wc-email-customer-invoice.php#L150) either. Thanks to @albarin for catching this 🙏

albarin reviewed (approved):
✅ Confirmed the different emails are sent with the expected info (change due, partial refund, and refund)
✅ Tested with different POS settings configuration.
